sdks/python/apache_beam/io/mongodbio.py

```diff
@@ -192,7 +192,7 @@ def get_range_tracker(self, start_position, stop_position):
   def read(self, range_tracker):
     with MongoClient(self.uri, **self.spec) as client:
       all_filters = self._merge_id_filter(range_tracker)
-      docs_cursor = client[self.db][self.coll].find(filter=all_filters)
+      docs_cursor = client[self.db][self.coll].find(filter=all_filters).sort("_id")
```

Comment:
if you don't want to fix the unit tests I think you can use .sort([('_id', ASCENDING)]) so it is more compliant with the fake cursor in unit test.